test/data/testinit.js

```diff
@@ -160,7 +165,9 @@ this.ajaxTest = function( title, expect, options ) {
 					if ( options.teardown ) {
 						options.teardown();
 					}
-					start();
+
+					// Make sure all events will be called before done()
+					setTimeout( done );
```

Comment:
This is done in case test body listens to ajax events, those events that do fire after `complete` should fire right after it